src/main/java/seedu/address/logic/commands/TagCreateCommand.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Creates a tag in the tagList.
+ */
+public class TagCreateCommand extends Command {
+
+    public static final String COMMAND_WORD = "create";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a tag to the address book. "
+            + "Parameters: "
+            + PREFIX_TAG + "TAG...\n"
+            + "Example: " + COMMAND_WORD + " "
```

Comment:
`"tag "` should appear before `COMMAND_WORD` for the full command

Reply:
Made the changes. Thanks.